mockwebserver/src/main/java/okhttp3/mockwebserver/RecordedRequest.kt

```diff
@@ -14,154 +14,102 @@
  * limitations under the License.
  */
 
-package okhttp3.mockwebserver;
+package okhttp3.mockwebserver
+
+import java.io.IOException
+import java.net.Inet6Address
+import java.net.Socket
+import javax.net.ssl.SSLSocket
+import okhttp3.Handshake
+import okhttp3.Headers
+import okhttp3.HttpUrl
+import okhttp3.TlsVersion
+import okio.Buffer
+
+/** An HTTP request that came into the mock web server.  */
+class RecordedRequest(
+  val requestLine: String?,
+  /** All headers. */
+  val headers: Headers?,
```

Comment:
non-null

Reply:
Line 948 in `KotlinSourceCompatibilityTest`:
`recordedRequest = RecordedRequest(null, null, null, 0L, null, 0, Socket())`
So I have to declare multiple constructors to be `nullable`,in this case should I modify `KotlinSourceCompatibilityTest`?

Reply:
Yes.

Reply:
Fixed.